packages/create-indiekit/package.json

```diff
@@ -19,7 +19,7 @@
   "type": "module",
   "main": "index.js",
   "bin": {
-    "create-indiekit": "bin/create.js"
+    "create-indiekit": "bin/create.js delete-me"
```

Comment:
What does `delete-me` do here? I need to revisit this, but I think this script is called when you run `npm init indiekit [directory name]` or `npm create indiekit [directory name]`. Does this interfere with this?

Reply:
Yes, I was using `delete-me` as a placeholder for `[directory name]`